app/views/devise/sessions/new.html.haml

```diff
@@ -24,7 +24,7 @@
       = t(:devise)[:sessions][:header]
     .form-group.login-buttons
       - if USE_SHIBBOLETH
-        = link_to t(:devise)[:sessions][:institution_users], identity_shibboleth_omniauth_authorize_path(service_request_id: @service_request.try(:id)), class: 'btn btn-primary omniauth-login-button'
+        = link_to t('devise.sessions.institution_users', institution: ENV['institution']), identity_shibboleth_omniauth_authorize_path(service_request_id: @service_request.try(:id)), class: 'btn btn-primary omniauth-login-button'
```

Comment:
@loganfreeman 
Can you add a default value for this to the `.env.example` file?

Reply:
added an .env.example:

institution=University.


if the environment variable is missing, the default value is just empty string.